### ISSUE TITLE
ffmpeg output is distorted (vertical lines)

### ISSUE THREAD

**Issue by kareiva** [rien/reStream#97]
I was using the reStream on a weekly basis and without any changes to the settings/code, the output became garbled.

![restream ffmpeg](https://github.com/rien/reStream/assets/3890253/39912b09-93b6-4112-9655-fea63414aaa4)

Host is Fedora release 38 (went from beta to GA over a few `dnf upgrade`s) 

I am currently at 3862d9c5c85d09c58e985f01049989f15a3d2830 and using the below:

```
$ ffmpeg -version
ffmpeg version 6.0 Copyright (c) 2000-2023 the FFmpeg developers
built with gcc 13 (GCC)
```

**Comment by Eeems** [comment]
What OS version is the device?

**Comment by kareiva** [comment]
Forgot to mention the remarkable2 is at 3.6.0.1865

**Comment by Eeems** [comment]
This is likely a duplicate of #95

**Comment by kareiva** [comment]
Thanks. I will try the suggested fix at
https://github.com/rien/reStream/issues/95#issuecomment-1681492697 and let
you know.

**Comment by kareiva** [comment]
Thanks! I think it was the pixel format in the first place that was causing the issue.

I have tuned my options to the below:

```
bytes_per_pixel=2
pixel_format="gray16be"
video_filters="curves=all='0/0 0.19/1 1/1',transpose=0"
```

Now, just a small leftover:
transposing with a flip makes the landscape mode turn to left, not to right as documented.

**Comment by FloranMartin** [comment]
Hello,
I got similar issue on Remarkable 2 - version 3.6.1.1894
It was solved with

In the header part of the code
line 15 : video_filters = "vflip"

In the part which check the remarkable 2 version  
line 146: width = 1872
line 147 : height = 2808
line 148 : bytes_per_pixel = 1
line 150 : pixel_format="grey12be" 
line 151 : video_filters = "$video_filters",transpose=2"

In my case the grey12be was cleaner but feel free to try out the available format in the list (ending by be is nicer than le).  

One remaining minor issue is that the stream is duplicated into two vertical windows but resizing the app window works just fine to remain lazy enough.

**Comment by rien** [comment]
This should be fixed in the latest release by #100. Let me know if you encounter any additional issues.
